### PR TITLE
Ubuntu package unix lists wrong config

### LINKED ISSUE #1092: Ubuntu package unix lists wrong config
```
Please ensure you create a configuration file at /etc/kanidm/config
An example is in /usr/share/kanidm-unixd/
Setting up kanidm (1.1.0-alpha.9-2022092911398e0238a) ...
```

Should be:

```
Please ensure you create a configuration file at /etc/kanidm/unixd and /etc/kanidm/config
An example is in /usr/share/kanidm-unixd/
Setting up kanidm (1.1.0-alpha.9-2022092911398e0238a) ...
```

### PR DESCRIPTION
**Pull request by yaleman**
Fixes #1092

Updates the unix package file and includes an example config

### CODE REVIEW

**Comment by yaleman** [comment]
Success! 😄

File: platform/debian/build_kanidm.sh [hunk 1]
```diff
@@ -46,8 +46,8 @@ if [ "$(which cargo | wc -l)" -eq 0 ]; then
     fi
 fi
 
-# this assumes the versions are in lock-step, which is fine at the moment.s
-KANIDM_VERSION="$(grep -ioE 'version.*' kanidm_tools/Cargo.toml | head -n1 | awk '{print $NF}' | tr -d '"')"
+# this assumes the versions are in lock-step, which is fine at the moment.
```
Comment (Firstyear) [inline platform/debian/build_kanidm.sh:49]: # Comments may be out of date 🤣

The root cargo.toml is the only one that sets a version with rust1.64 now :)

**Review by Firstyear — APPROVED** [review]

File: platform/debian/build_kanidm.sh (hunk 1 above)
Comment (yaleman) [reply platform/debian/build_kanidm.sh:49]: 🤣

Comment (Firstyear) [reply platform/debian/build_kanidm.sh:49]: Sorry for the SCREAMING it was meant to be a '#'

**Pull request merged**